packages/engine-client/src/EngineExecuteBuilder.ts

```diff
@@ -409,12 +409,12 @@ export class EngineExecuteBuilder {
   }
 
   protected async getTxNonceIfNeeded(
-    params: WithBaseEngineExecuteParams<unknown>,
+    params: WithBaseEngineExecuteParams<unknown> & { subaccountOwner?: string },
```

Comment:
Is it safe to enforce `{ subaccountOwner: string }`? Could this replace `unknown`?